TreeList: update Column Chooser and Header Filter demos (#2571)
DevExpress/devextreme-demos: JSDemos/Demos/TreeList/UsingHeaderFilter/description.md

### Enable Search Within Header Filters

> Users can search values within header filters. To display the search bar, assign **true** to the **search**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/search/#enabled) property in the global [headerFilter](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/) object or in a column's [headerFilter](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/headerFilter/) object. You can also specify a comparison operation used to search header filter values. For this, implement the **search**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/headerFilter/search/#mode) property.

Reviewer: ```suggestion
Users can search values within header filters. To display the search bar, assign **true** to the **search**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/search/#enabled) property in the global [headerFilter](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/) object or in a column's [headerFilter](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/headerFilter/) object. You can also specify a comparison operation used to search header filter values. Use the **search**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/headerFilter/search/#mode) property.
```
